transbank/webpay/webpayplus/exceptions/transaction_create_exception.py

```diff
@@ -0,0 +1,24 @@
+from .transbank_exception import TransbankException
+
+
+class TransactionCreateException(TransbankException):
+    DEFAULT_MESSAGE = 'Transaction could not be created. Please verify given parameters'
```

Comment:
This message should be more generic, it would point the integrator into looking for errors in their code when the exception something completely different.

You can try something similar to `Transaction could not be created. Unidentified error ` and try to pass the stacktrace so the integrator has information to report back to us